distributions/jamulus.desktop.in

```diff
@@ -4,7 +4,7 @@ Comment=Jam Session
 Comment[fr]=Séance de bœuf
 GenericName=Internet Jam Session Software
 GenericName[fr]=Logiciel de séance de bœuf sur Internet
-Exec=jamulus
+Exec=$$TARGET
```

Comment:
You should add the new file jamulus.desktop.in to the DISTFILES in the .pro file.

Reply:
Done - and the same for the .png.